templates/front_page.html

```diff
@@ -27,7 +27,7 @@
                     Projects
                 </div>
             </a>
-            <img src="{{ url_for('static', filename='projects_card_img.jpg')}}"
+           <a href="https://goo.gl/images/shx1pQ"></a>
                  class="block-img">
```

Comment:
Start tag missing (same in other three cards too)

Reply:
so what can i do to correct it.